examples/bin/cli.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+java -Xmx256m -Duser.timezone=UTC -Dfile.encoding=UTF-8 \
+        -cp lib/*:config/_common \
+        io.druid.cli.Main $* \
+                | grep -v " INFO \[main\] io.druid.initialization.Initialization "
```

Comment:
Would you tell me what `grep` is for?

Reply:
Would probably be nicer to do this via a log4j2-cli.xml designed for the CLI tools.

Reply:
@gianm I agree that makes sense. It will look cleaner.